.github/workflows/client-compatability-tests.yml

```diff
@@ -0,0 +1,411 @@
+          - name: ocr-besu
+            run: -run TestOCRBasic
+            file: ocr
+            client: besu
+            pyroscope_env: ci-smoke-ocr-evm-simulated        
+          - name: ocr-erigon
+            run: -run TestOCRBasic
+            file: ocr
+            client: erigon
+            pyroscope_env: ci-smoke-ocr-evm-simulated         
+          - name: ocr2-geth
+            run: -run TestOCRv2Basic
+            file: ocr2
+            client: geth
+            pyroscope_env: ci-smoke-ocr2-evm-simulated            
+          # Uncomment, when https://smartcontract-it.atlassian.net/browse/TT-753 is DONE            
+          # - name: ocr2-nethermind
+          #   run: -run TestOCRv2Basic
+          #   file: ocr2
+          #   client: nethermind
+          #   pyroscope_env: ci-smoke-ocr2-evm-simulated                         
+          - name: ocr2-besu
+            run: -run TestOCRv2Basic
+            file: ocr2
+            client: besu            
+            pyroscope_env: ci-smoke-ocr2-evm-simulated      
+          - name: ocr2-erigon
+            run: -run TestOCRv2Basic
+            file: ocr2
+            client: erigon            
+            pyroscope_env: ci-smoke-ocr2-evm-simulated                           
+    runs-on: ubuntu-latest
+    name: Client Compatability Test ${{ matrix.product.name }}
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.event.merge_group.head_sha }}
+      - name: Build Go Test Command
+        id: build-go-test-command
+        run: |
+          # if the matrix.product.run is set, use it for a different command
+          if [ "${{ matrix.product.run }}" != "" ]; then
+            echo "run_command=${{ matrix.product.run }} ./smoke/${{ matrix.product.file }}_test.go" >> "$GITHUB_OUTPUT"
+          else
+            echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
+          fi
+      - name: Check for "enable tracing" label
+        id: check-label
+        run: |
+          label=$(jq -r '.pull_request.labels[]?.name // empty' "$GITHUB_EVENT_PATH")
+
+          if [[ -n "$label" ]]; then
+            if [[ "$label" == "enable tracing" ]]; then
+              echo "Enable tracing label found."
+              echo "trace=true" >> $GITHUB_OUTPUT
+            else
+              echo "Enable tracing label not found."
+              echo "trace=false" >> $GITHUB_OUTPUT
+            fi
+          else
+            echo "No labels present or labels are null."
+            echo "trace=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Setup Grafana and OpenTelemetry
+        id: docker-setup
+        if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.product.tag_suffix == '-plugins'
```

Comment:
just curious, what's the reason for doing that only for plugins?

Reply:
This is left-over from some of core's test monitoring stack, can probably rip this out actually